internal/service/chat.go

```diff
@@ -89,6 +89,14 @@ func (s Chat) AcceptInvitation(invitationCode, userUUID string) error {
 	return s.channelManager.AddMember(channel, user, "")
 }
 
+func (s Chat) IsMemberOfChannel(channelUUID, userUUID string) (bool, error) {
+	_, err := s.channelManager.GetMemberInfo(channelUUID, userUUID)
+	if err != nil {
+		return false, err
+	}
+	return true, err
+}
```

Comment:
I don't like this. Perhaps it would be better to just return MemberInfo insead. Will inspect this further when dealing with unread status.